### PR TITLE
Navbar component

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added Navbar Component

Live Preview : https://framer-ui.netlify.app/docs/navigation.html

### CODE REVIEW

File: components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* @import url('../../components/root.css');
+@import url('../../components/list/list.css');
+@import url('../../components/toast/toast.css'); */
+
+@import url('../../components/framer.css');
+
+.navbar{
+    padding:1rem;
+    background: white;
+    color: rgb(22, 21, 21);
```
Comment (rahulk31) [inline components/navbar/navbar.css]: Since this is a component, may be you can use color variable here.

File: components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+.navbar-dark-text{
+    color: var(--col-dark);
+}
+
+.navbar-icons{
+    padding: 0.5rem;
+    font-size: 1.3rem !important;
+    cursor: pointer;
+}
```
Comment (rahulk31) [inline components/navbar/navbar.css]: Suggestion: To avoid `!important` tags.
Depends on the usecase, if you are trying to force the color then its fine 👍

**Review by rahulk31 — APPROVED** [review]
HTML code is well structured. Overall looks good to be merged.

File: docs/navigation.html [hunk 3]
```diff
@@ -0,0 +1,460 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navbars</title>
+    <link rel="stylesheet" href="../components/navbar/navbar.css">
+    <meta name="theme-color" content="#2C2929">
+
```
Comment (Ankur9669) [inline docs/navigation.html:11]: Question: I am not sure about this content

File: docs/navigation.html [hunk 4]
```diff
@@ -0,0 +1,460 @@
...
+                                        <span class="font-bold">Zeplin</span>
+                                    </li>
+                                </a>
+                            </ul>
+                        </div>
+
+                        <div class="flex flex-space-evenly flex-align-item-center ">
+                            <ul class=" text-deocration-none flex flex-wrap gap-col-2">
+                                <a href="#" class="text-deocration-none" target="_blank" rel="noreferrer">
+                                    <li class="pb-1 nav-text navbar-dark-text flex flex-align-item-center gap-1">
```
Comment (Ankur9669) [inline docs/navigation.html:191]: fix: These links would not be target=blank beacause in navbar we want to navigate within our app

**Review by Ankur9669 — APPROVED** [review]
Code and UI looks great. Nice Work!!

File: docs/navigation.html (hunk 3 above)
Comment (mehulsatardekar) [reply docs/navigation.html:11]: > Question: I am not sure about this content

This meta tag add color to the url header on mobile devices.
Screenshot_20220220-201457

File: docs/navigation.html (hunk 4 above)
Comment (mehulsatardekar) [reply docs/navigation.html:191]: sure will make the changes

**Review by j24m — APPROVED** [review]
I can see that your indentations are not consistent  throughout, please make sure to improve this, reason is, when you start working somewhere and there are big projects which have  huge lines of code, the code will look ugly and unreadable.

File: docs/navigation.html [hunk 5]
```diff
@@ -0,0 +1,460 @@
...
+                </a>  
+                <a href="https://www.youtube.com/channel/UCOPdSGIn44Gh4QXZH_6U8gQ"  target="_blank" rel="noreferrer">
+                    <box-icon name='youtube' type='logo' color='#eeeded' ></box-icon>  
+                </a> 
+            </div>
+        </div>
+        <aside class="sidebar-menu  flex flex-column flex-between"  id="sidebar-menu">
+            <div class="flex flex-column flex-between mt-1">
+                <div class="flex flex-between flex-align-item-center ">
+                    <span class="font-mid-bold ml-1 logo-text">Framer-UI</span>
```
Comment (rahulyadav139) [inline docs/navigation.html:37]: You have used utility classes extensively which is great but you have named these classes a little bit understandable.  Navbar is looking great. Codes are clean but somewhere indentation is an issue. Other than that everything is great. Great work.

**Review by rahulyadav139 — APPROVED** [review]

File: docs/navigation.html (hunk 4 above)
Comment (mehulsatardekar) [reply docs/navigation.html:191]: > sure will make the changes



> fix: These links would not be target=blank beacause in navbar we want to navigate within our app

well @Ankur9669 its upto the user if they want to remove it they can remove it when they  implement in their app

**Pull request merged**